Cargo.toml

```
@@ -21,8 +21,7 @@ backon = { version = "1.5.1", features = ["tokio-sleep"] }
 borsh = { version = "1.5.1", features = ["derive"] }
 clap = { version = "4.5.20", features = ["derive", "env"] }
 curve25519-dalek = "4.1.3"
-# TODO(#815): Use upstream library
-dcap-qvl = { git = "https://github.com/gilcu3/dcap-qvl", rev = "5a264b50dbe833b807d4ed6dfff6d9ef838cd024", default-features = false, features=["contract", "std", "report"] }
+dcap-qvl = { git = "https://github.com/Phala-Network/dcap-qvl", rev = "06ac37d3fd7f9690158d92025666eb3d1e3f1938", default-features = false, features=["contract", "borsh", "std"] }
```

Nit: It's nice to explain where these refs come from.
```suggestion
// Lastest `master` as per 2025-08-15
dcap-qvl = { git = "https://github.com/Phala-Network/dcap-qvl", rev = "06ac37d3fd7f9690158d92025666eb3d1e3f1938", default-features = false, features=["contract", "borsh", "std"] }
```